### PR TITLE
feat: update performance plotting scripts

### PR DESCRIPTION
**Pull request by asalzburger**
This updates the performance scripts as suggested.

### CODE REVIEW

File: Examples/Scripts/TrackingPerformance/reconstructionPerformance.C [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    emho[itag] =
+        new TCanvas(name.c_str(), label.c_str(), itag * 10, 10, 1800, 300);
+    emho[itag]->Divide(5, 1);
+    ++itag;
+  }
+
+  unsigned int imode = 1;
+  for (auto fileName : inputFileNames) {
+    // Te approprate file
+    auto file = TFile::Open(fileName.c_str(), "read");
```
Comment (XiaocongAi) [inline Examples/Scripts/TrackingPerformance/reconstructionPerformance.C:41]: Shall we throw a failure if `file == nullptr`?

File: Examples/Scripts/TrackingPerformance/CommonUtils.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// Copyright (C) 2019-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <TColor.h>
+
+// Helper function:
+// function to set up the histgram style
```
Comment (XiaocongAi) [inline Examples/Scripts/TrackingPerformance/CommonUtils.h:12]:
```suggestion
// function to set up the histogram style
```

File: Examples/Scripts/TrackingPerformance/CommonUtils.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  hist->SetMarkerStyle(20);
+  hist->SetMarkerSize(0.8);
+  hist->SetLineWidth(2);
+  // hist->SetTitle("");
+  hist->SetLineColor(color);
+  hist->SetMarkerColor(color);
+}
+
+// Helper function:
+// function to set up the efficiency histgram style
```
Comment (XiaocongAi) [inline Examples/Scripts/TrackingPerformance/CommonUtils.h:31]:
```suggestion
// function to set up the efficiency histogram style
```

**Review by XiaocongAi — APPROVED** [review]
Hi @asalzburger , thank you for implementing those plotting functions. They look all good except a minor comment you might want to address.
On a side note, I think in the longer term it might be better for us to make the performance plots based on the persistified tracks in the root files since this could gives us more flexibility regarding the definition of the performance, the  binning of the variables etc. and more importantly, we don't have to redo the reconstruction just in order to remake the performance plots.

File: Examples/Scripts/TrackingPerformance/reconstructionPerformance.C (hunk 1 above)
Comment (asalzburger) [reply Examples/Scripts/TrackingPerformance/reconstructionPerformance.C:41]: I will do this in another iteration.

**Pull request merged**